aten/src/ATen/native/cuda/Normalization.cu

```diff
@@ -722,6 +722,16 @@ std::tuple<Tensor, Tensor> batch_norm_update_stats_cuda(
   c10::MaybeOwned<Tensor> running_var = at::borrow_from_optional_tensor(running_var_opt);
 
   const int64_t n_input = self.size(1);
+
+  TORCH_CHECK(
+      self.size(0) > 0 && n_input > 0 && self.numel() > 0,
+      "input.size(0), input.size(1) and input.numel() need to be greater than 0, got input.size(0) = ",
+      self.size(0),
+      ", input.size(1) = ",
+      n_input,
+      ", input.numel() = ",
+      self.numel()
+  );
```

Comment:
Same as the above, use Occam's razor and device a simplest check (and perhaps error message)

Reply:
Modified.